packages/css/src/types.ts

```diff
@@ -808,4 +808,14 @@ export interface Theme {
    * @see https://theme-ui.com/components/message#variants
    */
   messages?: Record<string, ThemeUIStyleObject>
+
+  /**
+   * When the theme values are replaced with css custom vars, save original values
+   */
+  original?: {
```

Comment:
I'm skeptical towards including it in the types for our public API.
1. Users won't ever add it manually. If they have a key called original, we'd overwrite it.
2. We don't have that value on the theme in **our context**. We add it to theme's shallow copy passed to **Emotion's context**.

Reply:
removed